spinalcordtoolbox/utils/fs.py

```diff
@@ -151,12 +151,11 @@ def cleanup(self):
         rmtree(self.path_tmp)
 
 
-def cache_signature(input_files=[], input_data=[], input_params={}):
+def cache_signature(input_files=[], input_params={}):
```

Comment:
This one is also a change of behaviour, but no callers use the parameter `input_data`, and the code for handling it was unconditionally broken.

Reply:
Thanks for the sleuthing and extra context!